doc/nrf/device_guides/working_with_nrf/nrf52/developing.rst

```diff
@@ -82,6 +82,45 @@ If you want to add SMP Service to advertising data, refer to the :ref:`zephyr:sm
 
 .. fota_upgrades_over_ble_additional_information_end
 
+FOTA updates are also supported when MCUboot is in Direct-XIP mode.
```

Comment:
consider putting labels at the top and the bottom so that we can declare support for the direct-xip in case we add it for other platforms that are documented in their respective "Working with..." sections